src/App.tsx

```diff
@@ -76,18 +67,15 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !hasError && posts.length === 0 && (
```

Comment:
Use logical operators 
```suggestion
                {author && loading && !hasError && !posts.length && (
```